installer/manifests/keptn/values.yaml

```
@@ -14,11 +14,15 @@ mongo:
   host: mongodb:27017
   ## @param mongo.architecture
   architecture: standalone
+  updateStrategy:
+    ## @param mongo.updateStrategy.type
```

You could already add some small documentation

```suggestion
    ## @param mongo.updateStrategy.type Set the update strategy for MongoDB
```